test/index.test.js

```diff
@@ -37,3 +40,51 @@ describe('app', () => {
   })
 
 })
+
+describe ('gateway', () => {
```

Comment:
This belongs in a separate test suite like `gateway.test.js`.

Reply:
OK, I'm on it!